custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -24,10 +31,13 @@
     disease_classification,
     dcexpulmonory,
     dcpulmonory,
+    hiv_status,
+    art_initiated,
 )
 from custom.enikshay.case_utils import update_case
 
 ENIKSHAY_ID = 8
+NIKSHAY_NULL_DATE = '1990-01-01'
```

Comment:
Actually it should be 1900 only. Nikshay wants a value which is way before so its not valid. This date was suggested by them only.